test/integration/targets/vmware_guest/tasks/poweroff_d1_c1_f0.yml

```diff
@@ -48,4 +48,4 @@
 - name: make sure no changes were made
   assert:
     that:
-        - "poweroff_d1_c1_f0.results|map(attribute='changed')|unique|list == [False]"
+        - "poweroff_d1_c1_f0.results|map(attribute='changed')|unique|list == [True]"
```

Comment:
Why is the task name `make sure no changes were made` when we're asserting that changes were made?

Reply:
In previous VCSIM version, vms were spawned in "poweredoff"  state, and running this PB would poweroff them. But now with latest changes in VCSIM, vms are spawned in "PoweredOn" mode so running the PB would make "changed" as True. I agree to change task name as it is confusing.

Reply:
@Akasurde Thanks for the explanation.
@dericcrago This PR should be ready to merge once the task names are fixed.

Reply:
thanks @Akasurde & @mattclay - the task names have been updated and a comment has also been added to hopefully help out in the future